XenModel/FriendlyNames.resx

```diff
@@ -546,18 +582,45 @@
   <data name="Label-host.edition-advanced" xml:space="preserve">
     <value>[XenServer product] Advanced Edition</value>
   </data>
+  <data name="Label-host.edition-desktop" xml:space="preserve">

```

Comment:
There are a lot of new strings in this file, and they don't appear to be all new or recent features. Where did they come from? We need to understand this before we can merge them.